docs/api/covidcast-signals/changehc.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Change Healthcare
+parent: Data Sources and Signals
+grand_parent: COVIDcast Epidata API
+---
+
+# Change Healthcare
+{: .no_toc}
+
+* **Source name:** `changehc`
+* **Number of data revisions since 19 May 2020:** 0
+* **Date of last change:** Never
+* **Available for:** county, hrr, msa, state (see [geography coding docs](../covidcast_geography.md))
+
+This data source is based on information about outpatient visits, provided to us
+by Change Healthcare. Using this outpatient data, we estimate the
```

Comment:
This should say what Change Healthcare is. A reader who isn't familiar will not know that they process claims on behalf of others, for example, and that's useful context for understanding what will and won't be included in the data

Reply:
How is the current description?

Reply:
Looks good, but I agree Roni should probably check that the description is accurate and won't cause us problems. You could ping him and show him the draft

Reply:
Looks good, but I agree Roni should probably check that the description is accurate and won't cause us problems. You could ping him and show him the draft